elk/kibana/config/kibana.yml

```
@@ -31,70 +31,70 @@ elasticsearch.password: ${KIBANA_SYSTEM_PASSWORD}
-#        package:
-#          name: apm
-#        # See the APM package manifest for a list of possible inputs.
-#        # https://github.com/elastic/apm-server/blob/v8.5.0/apmpackage/apm/manifest.yml#L41-L168
-#        inputs:
-#          - type: apm
-#            vars:
-#              - name: host
-#                value: 0.0.0.0:8200
-#              - name: url
-#                value: http://apm-server:8200
+xpack.fleet.agents.fleet_server.hosts: [ http://fleet-server:8220 ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)